src/main/java/edu/kit/provideq/toolbox/featuremodel/anomaly/dead/DeadFeatureConfiguration.java

```diff
@@ -0,0 +1,56 @@
+package edu.kit.provideq.toolbox.featuremodel.anomaly.dead;
+
+import edu.kit.provideq.toolbox.ResourceProvider;
+import edu.kit.provideq.toolbox.meta.Problem;
+import edu.kit.provideq.toolbox.meta.ProblemManager;
+import edu.kit.provideq.toolbox.meta.ProblemType;
+import java.io.IOException;
+import java.util.Objects;
+import java.util.Set;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Definition and registration of the "dead feature" feature model anomaly problem.
+ */
+@Configuration
+public class DeadFeatureConfiguration {
+  /**
+   * A searching problem:
+   * For a given feature model, check if the model contains dead features.
+   *
+   * @see <a href="https://sdq.kastel.kit.edu/publications/pdfs/kowal2016b.pdf">
+   *      "Explaining Anomalies in Feature Models", Kowal et al., 2026</a>
+   */
+  public static final ProblemType<String, String> FEATURE_MODEL_ANOMALY_DEAD = new ProblemType<>(
+      "feature-model-anomaly-dead",
+      String.class,
+      String.class
+  );
+
+  @Bean
+  ProblemManager<String, String> getDeadFeatureManager(
+      SatBasedDeadFeatureSolver satSolver,
+      ResourceProvider resourceProvider
+  ) {
+    return new ProblemManager<>(
+        FEATURE_MODEL_ANOMALY_DEAD,
+        Set.of(satSolver),
```

Comment:
Since my time in the ProvideQ project is coming to an end, I would rather like to focus on completing the numerous changes required to make the problem graph work with a good architecture. I agree that this is a good idea but implementing it would require a bit more thought into: whether all problem solvers should be registered as spring components, whether the solvers can be fetched once at startup or if they need to be fetched continuously, and quite possibly further considerations. If you insist, I can include this into this pull request, but as this is not required to complete the problem graph, I would prefer deferring this to a different PR

Reply:
I don't insist 😄 
We can best make an issue for this. #72